index.html

```diff
@@ -1430,9 +1441,50 @@
 
 ## Harassment
 
-<aside class="issue">
-  This section has not been written.
-</aside>
+Online <dfn>harassment</dfn> is the "pervasive or severe targeting of an individual or group online
+through harmful behavior" [[PEN-Harassment]]. Harassment is a prevalent problem on the Web,
+particularly via social media. Harassment may be more severe and its consequences more impactful for
+LGBTQ people and women [[?PEW-Harassment]].
+
+[=Harassment=] is both a violation of privacy itself and can be magnified or facilitated by other
+violations of privacy. 
+
+Abusive online behavior may include: sending [=unwanted information=]; directing others to contact
+or bother a person ("dogpiling"); disclosing sensitive information about a person; posting false
+information about a person; impersonating a person; insults; threats; and hateful or demeaning
+speech.
+
+Disclosure of identifying or contact information (including "doxxing") can be used, including by a
+separate attacker, to send often persistent unwanted information that amounts to harassment.
+Disclosure of location information can be used, including by a separate attacker, to intrude on a
+person's physical safety or space.
+
+Mitigations for harassment include but extend beyond mitigations for unwanted information and other
+privacy principles. Harassment can include harmful activity not solely directly communicated to the
```

Comment:
"solely directly communicated" is a bit obscure. "... with a wider distribution than just the target ..."?